core/routemgmt/getApi/getApi.js

```diff
@@ -154,12 +154,12 @@ function validateArgs(message) {
     return 'Internal error. A message parameter was not supplied.';
   }
 
-  if (!message.__ow_user) {
```

Comment:
Right.  Each action has it's own validateArgs function and all are slightly different.